packages/smarthr-ui/src/components/NotificationBar/NotificationBar.tsx

```diff
@@ -142,7 +146,7 @@ export const NotificationBar: React.FC<Props & ElementProps & BaseProps> = ({
   message,
   onClose,
   children,
-  role = type === 'info' ? 'status' : 'alert',
+  role = type.match(/info|sync/) ? 'status' : 'alert',
```

Comment:
```suggestion
  role = type.match(/^(info|sync)$/) ? 'status' : 'alert',
```

細かい話ですが完全一致でないのは若干気になりがあるのでこんな感じどうでしょうか！

Reply:
ありがとうございます!